cards.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
```

Comment:
Include more comments, especially when your code doesn't work. It seems like you got the basic bones of the program done correctly, just didn't quite get to finish it off